genie-client/src/main/java/com/netflix/genie/client/interceptors/ResponseMappingInterceptor.java

```diff
@@ -55,6 +56,10 @@ public Response intercept(final Chain chain) throws IOException {
         if (response.isSuccessful()) {
             return response;
         } else {
+            if (response.code() == 429) {
```

Comment:
there should be a constant from either okhttp or UrlConnection you can use for this number.

Reply:
you'd be surprised 429 is not on those lists, unless i am mistaken.

Reply:
haha wow. All right, you can make it your own constant if you want but not a huge deal